### ISSUE TITLE
[Accessibility]: use image descriptions (alt text)

### ISSUE THREAD

**Issue by christopherpross** [Bootstrap-Academy/Bootstrap-Academy#82]
### Bootstrap Academy username

cpross

### Contact Details

cpross@cpross:matrix.org

### related to

#74 

### What browsers are you seeing the problem on?

Firefox, Chrome, Microsoft Edge

### What happened?

Hey Folks,

I hope you're all doing well! I wanted to bring up an important point about our website's accessibility that we could improve together.

## What's missing?

It looks like several images on our site, e.g. those of our sponsors, don't have alt text descriptions. This means folks who use screen readers, as well as users who have images turned off, might be missing out on important info.

## Why does this matter?

Alt text is a key part of web accessibility, making sure that everyone can get the full story of our webpage, regardless of how they interact with it. Plus, it's a part of inclusive design which, I believe, is something we all support.

### 1. Add alt text to images:
Every image should have alt text that describes what's being displayed. This is crucial for images that contain information, such as the logos for our sponsors. Here’s an example:

```html
<img src="sponsor-logo-BWB.png" alt="Bundeswettbewerb künstliche Intelligenz - supporting Bootstrap Academy">
```

### 2. Educate on writing effective alt text:
Let’s craft a mini-guide for writing good alt text. Good alt text should be:

- Concise and specific: Short and to the point, but descriptive enough to convey the image’s purpose.
- Informative: It should reflect the image's content and function.
- Non-redundant: Avoid stating "image of..." or "graphic of..." as screen readers identify images already.
- Context-aware: The description should be appropriate to the image's context within the content.
- Text replicative: If the image has text, the alt text should include the same words.

We could share these pointers with our contributors, maybe in the README or in our contribution guidelines?
### 3. Ensure future images include alt text:
As we continue to develop our site, let's ensure all new images include alt text right from the get-go. This proactive approach will save us the hassle of backtracking and help keep our site accessible with each update. Moreover, with AI tools now available, generating descriptive and accurate alt texts can be quick and straightforward. This doesn't just apply to future images; let's also encourage everyone contributing to the codebase to use these tools to add or improve alt texts.

## Wrapping up:
Making our site more accessible is a win-win for everyone. It'd be great to get your thoughts or have some of you jump in and help with this effort.
Thanks for your time and all the fantastic work you're doing! Let's make Bootstrap Academy a place for everyone.
Cheers,
Christopher
